src/LSF.jl

```diff
@@ -1,11 +1,12 @@
 normal_pdf(Δ, σ) = exp(-0.5*Δ^2 / σ^2) / √(2π) / σ
 
 """
-Returns the matrix which transforms a spectrum to its LSF-convolved version for spectral resolution 
+Returns the matrix which a gaussian line spread function with constant spectral resolution, 
```

Comment:
I don't mean to nitpick, but there's a typo here

Reply:
Not only that, it was factually wrong!  thanks.